src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```

Comment:
Color and number are characteristics of a ball, as they are fields of the `Ball` class. So you can name them simply `color` and `number`.
```suggestion
    private String color;
    private int number;
```